runtime/libffi/z/sysvz64.s

```diff
@@ -679,16 +615,17 @@ STFPCHKD246 DS 0H
 *determine how to pass the struct based on size
 *at this point we've weeded out all the float/double
 *pair structs that get passed purely in FPRs
-STRCT2  DS 0H
-         LG 15,0(,11)          type->size
-         CFI 15,8              Struct <= 8 bytes?
+STRCT    DS 0H
+         LG 11,0(10,5)
+         LG  15,0(,11)          type->size
```

Comment:
The operands on line 619 should align with those on lines 620-621 (one more space).

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/compare/91cb2ada99ecad623c87439a68e9c15758d5f34a..5ac3c7c654f2169e7a1df18432506fe976fe3fe5